lib/src/drawings.dart

```diff
@@ -158,8 +154,8 @@ class TextDrawing extends GYWDrawing {
     controlBytes.add(int16Bytes(left));
     controlBytes.add(int16Bytes(top));
 
-    controlBytes.add(utf8.encode(font?.prefix ?? "NUL"));
-    controlBytes.add(int8Bytes(size ?? 0));
+    controlBytes.add(utf8.encode(font.filename.padRight(5, "\u0000")));
```

Comment:
I'm scared of allowing padded font, can't we use an assert in the constructor or something similar ? isn't it safer and easier to handle

Reply:
It's not necessarily safer because I always add a null terminator when I parse the command, so it's the same. But sure. I'm impartial.

Reply:
> It's not necessarily safer because I always add a null terminator when I parse the command,

That's a good a safe way to manage it on the firmware side. Let's still add the `assert` here to discourage the user to do so

Reply:
yes, it's fine. Just add the error message related to the assert. I think there is a rule in `flutter_lints` that prevents to write assert without the associate message, can you enable it ?